packages/libtss2/tss-sysusers.conf

```diff
@@ -0,0 +1,2 @@
+# User for TPM devices
+u tss 59
```

Comment:
Was this user selected to match what other Linux distros use?

Reply:
> Was this user selected to match what other Linux distros use?

Yes, I picked the same one as Fedora.